nip47/publish_nip47_info.go

```diff
@@ -37,7 +38,7 @@ func (svc *nip47Service) GetNip47Info(ctx context.Context, relay *nostr.Relay, a
 func (svc *nip47Service) PublishNip47Info(ctx context.Context, relay nostrmodels.Relay, appWalletPubKey string, appWalletPrivKey string, lnClient lnclient.LNClient) (*nostr.Event, error) {
 	var capabilities []string
 	var permitsNotifications bool
-	tags := nostr.Tags{[]string{}}
+	tags := nostr.Tags{[]string{"v", cipher.SUPPORTED_VERSIONS}}
```

Comment:
I had to do this to avoid an import cycle (`cipher` imports `models` which imports `transactions` which imports `tests` which imports `cipher` again)

> `models` which imports `transactions`

I'm adding a commit which fixes the imports as this part doesn't make sense